Procfile

```diff
@@ -1,4 +1,5 @@
-db: docker  run -v ${PWD}/scripts/init_mongo.js:/docker-entrypoint-initdb.d/init_mongo.js:ro --env-file ${PWD}/.env -p 27017:27017 -p 28017:28017 mongo
+db:             docker run -v ${PWD}/scripts/init_mongo.js:/docker-entrypoint-initdb.d/init_mongo.js:ro --env-file ${PWD}/.env -p 27017:27017 -p 28017:28017 mongo
+messagebroker:  docker run -v ${PWD}/scripts/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf:ro -v ${PWD}/scripts/rabbitmq-definitions.json:/etc/rabbitmq/rabbitmq-definitions.json:ro -v mq:/var/lib/rabbitmq --env-file ${PWD}/.env -p 15672:15672 -p 5672:5672 rabbitmq:3.11.1-management-alpine
```

Comment:
not sure the `.env` file will work, because if it will contain the same values as `.env.example` then i ti is missing `RABBITMQ_DEFAULT_USER`, `RABBITMQ_DEFAULT_PASS` and `RABBITMQ_CONFIG_FILE`